spec-grpc/src/main/java/io/a2a/grpc/utils/ProtoUtils.java

```diff
@@ -182,8 +188,8 @@ public static io.a2a.grpc.TaskArtifactUpdateEvent taskArtifactUpdateEvent(TaskAr
             builder.setTaskId(event.getTaskId());
             builder.setContextId(event.getContextId());
             builder.setArtifact(artifact(event.getArtifact()));
-            builder.setAppend(event.isAppend() == null ? false : event.isAppend());
-            builder.setLastChunk(event.isLastChunk() == null ? false : event.isLastChunk());
+            builder.setAppend(requireNonNullElse(event.isAppend(), false));
```

Comment:
Same thing here, I think we can just call the method if non null and let the builder set the right default.